### PR TITLE
[WIP] Move _cmbtp_standard logic from records.py to functions.py file

### PR DESCRIPTION
**Pull request by martinholmer**
This pull request continues making changes that move toward the goal of eliminating the `Records.impute_variables()` method as described in issue #688.  But instead of moving the calculation of `_cmbtp_standard` into the taxdata final-prep script, the logic is moved into the functions.py file, which used the `_cmbtp_standard` variable just once in the `AMTI()` function.

The expectation was that making this logic change would have no effect on unit test results, but it turned out to produce very minor changes in the results of the two requires_pufcsv unit tests.  The reason for these changes is that moving the `_cmbtp_standard` logic involved fixing a problem in the master branch code: the `e00100` variable was not being "aged" in the `Records._blowup()` method.

@MattHJensen @feenberg @Amy-Xu @GoFroggyRun

### CODE REVIEW

**Comment by MattHJensen** [comment]
Thanks @martinholmer. I'll take a look at this later today.

**Comment by GoFroggyRun** [comment]
Thanks @martinholme, these changes look good to me.

**Comment by MattHJensen** [comment]
Still considering whether this works. Hopefully I'll figure it out soon. _cmbtp is a min imputation of amt preferences.

**Comment by martinholmer** [comment]
Will transform pull request #721 into two: (1) blowup-bug-fix and (2) impute-logic-move.  That will demonstrate that the impute-logic-move causes no changes in test results after the blowup-bug-fix has been implemented.

File: taxcalc/functions.py [hunk 1]
```diff
@@ -824,7 +824,7 @@ def AMTI(c60000, _exact, e60290, _posagi, e07300, c24517,
         c62100 += _cmbtp
     if puf and _standard > 0:
         if f6251 == 1:
-            _cmbtp = _cmbtp_standard
+            _cmbtp = e62100 - e00100 + e00700
```
Comment (MattHJensen) [inline taxcalc/functions.py]: @martinholmer, I believe AMT preferences (`_cmbtp`) ought to be imputed with the base year's data and then blown up separately. Therefore, moving the imputation for standard deduction filers functions.py here is not correct. I think we ought to treat _cmbtp like all of our other imputations: do the imputation in TaxData and then include the starting year values in puf.csv.

**Comment by martinholmer** [comment]
Retract pull request #721 in favor of future pull request that moves imputation of the _cmbtp_standard (and _cmbtp_itemizer) variable from the Records class to the taxdata repository.

**Pull request closed without merging**